### PR TITLE
Add project list to organization reports.

### PR DESCRIPTION
**Pull request by tkaemming**
This could probably use some cleanup, but it's more or less stable.

References GH-3945.

@getsentry/platform @ckj

---

This change is [Reviewable](https://reviewable.io/reviews/getsentry/sentry/4156)

### CODE REVIEW

File: src/sentry/tasks/reports.py [hunk 1]
```diff
@@ -638,46 +638,118 @@ def deliver_organization_user_report(timestamp, duration, organization_id, user_
...
+    # Starting building the list of items to include in the report chart. This
+    # is a list of [Key, Report] pairs, in *ascending* order of the total sum
+    # of values in the series. (This is so when we render the series, the
+    # largest color blocks are at the bottom and it feels appropriately
+    # weighted.)
+    selections = map(
+        lambda (instance, color): (
+            Key(
+                u'{} / {}'.format(
+                    instance.team.name,
```
Comment (dcramer) [inline src/sentry/tasks/reports.py]: I think we should just use ``instance.slug`` here (continue to deprecate team names).

**Review by dcramer — APPROVED** [review]
🚀

File: src/sentry/tasks/reports.py (hunk 1 above)
Comment (tkaemming) [inline src/sentry/tasks/reports.py]: I think dropping team name is fine. Why `slug` instead of `name`?

Comment (dcramer) [inline src/sentry/tasks/reports.py]: We could use name, but the nice thing about slug is its unique. I think its probably worthwhile for us (over time) to push it as the primary name for items.

In many cases we already do ``{org slug} / {project slug}`` to identify projects, giving it a GitHub-esque feel.

**Comment by ckj** [comment]
:metal:

**Pull request merged**